src/ansys/simai/core/data/training_data.py

```diff
@@ -97,13 +98,28 @@ def get_subset(self, project: Identifiable["Project"]) -> Optional[SubsetEnum]:
     def assign_subset(
         self, project: Identifiable["Project"], subset: Optional[Union[SubsetEnum, str]]
     ) -> None:
-        """Assign the training data subset in relation to a given project.
+        """Assign the training data to a subset in relation to a given project.
 
         Args:
             project: ID or :class:`model <.projects.Project>`
             subset: SubsetEnum attribute (e.g. SubsetEnum.TRAINING) or string value (e.g. "Training") or None to unassign.
```

Comment:
General question: Could we only give the enum option instead of both enum and string as the string option is error-prone. What flexibility does the string offer the user?

Reply:
We could. But as a developer typing a string is more convenient than importing and using an Enum albeit more error prone.